<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcDataWdg</class>
 <widget class="QWidget" name="ProcDataWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Program</string>
     </property>
     <property name="buddy">
      <cstring>programLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="programLineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="programButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Arguments</string>
     </property>
     <property name="buddy">
      <cstring>argumentsListWidget</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QListWidget" name="argumentsListWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Program Input</string>
     </property>
     <property name="buddy">
      <cstring>inputListWidget</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QListWidget" name="inputListWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="wrkDirLabel">
     <property name="text">
      <string>Working directory</string>
     </property>
     <property name="buddy">
      <cstring>wrkDirLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="wrkDirLineEdit"/>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="wrkDirButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>programLineEdit</tabstop>
  <tabstop>argumentsListWidget</tabstop>
  <tabstop>inputListWidget</tabstop>
  <tabstop>wrkDirLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
